src/main/java/dk/cs/aau/huppaal/presentations/LogTabPresentation.java

```diff
@@ -0,0 +1,175 @@
+package dk.cs.aau.huppaal.presentations;
+
+import dk.cs.aau.huppaal.BuildConfig;
+import dk.cs.aau.huppaal.HUPPAAL;
+import dk.cs.aau.huppaal.controllers.LogTabController;
+import dk.cs.aau.huppaal.logging.Log;
+import dk.cs.aau.huppaal.logging.LogRegex;
+import dk.cs.aau.huppaal.logging.LogLinkQuantifier;
+import dk.cs.aau.huppaal.presentations.logging.Hyperlink;
+import dk.cs.aau.huppaal.presentations.logging.HyperlinkTextArea;
+import dk.cs.aau.huppaal.presentations.logging.TextStyle;
+import dk.cs.aau.huppaal.presentations.util.PresentationFxmlLoader;
+import dk.cs.aau.huppaal.utility.helpers.SelectHelper;
+import javafx.application.Platform;
+import javafx.beans.NamedArg;
+import javafx.beans.value.ChangeListener;
+import javafx.scene.control.Tooltip;
+import javafx.scene.layout.HBox;
+import org.fxmisc.flowless.VirtualizedScrollPane;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.net.URI;
+import java.util.Optional;
+
+public class LogTabPresentation extends HBox {
+    public LogTabController controller;
+    public boolean autoscroll, wordwrap;
+    private final HyperlinkTextArea logArea;
+    private VirtualizedScrollPane<HyperlinkTextArea> scrollPane;
+    public LogTabPresentation(@NamedArg("textColor") String textColor) {
+        controller = PresentationFxmlLoader.loadSetRoot("LogTabPresentation.fxml", this);
+        wordwrap = true;
+        autoscroll = true;
+        logArea = new HyperlinkTextArea(this::onLinkClick);
+        scrollPane = new VirtualizedScrollPane<>(logArea);
+        initializeLogArea(textColor);
+        initializeButtons();
+        setupWordWrap();
+        setupAutoscroll();
+        Log.addOnLogAddedListener(this::onLogAdded);
+    }
+
+    private void onLinkClick(Hyperlink link) {
+        try {
+            var matcher = LogRegex.PATTERN.matcher(link.getLink());
+            if(!matcher.find()) // *should* never happen
+                throw new RuntimeException("Not a valid %s link pattern: '%s'".formatted(BuildConfig.NAME,link.getLink()));
+            // TODO: When syntactic elements have UUIDs to identify them, we should look at those IDs (instead/aswell)
+            var ref  = matcher.group("ref");
+            var ref1 = matcher.group("ref1");
+            var ref2 = Optional.ofNullable(matcher.group("ref2"));
```

Comment:
Consider renaming these to something more descriptive. I assume that they hold different sets of the referenced elements. And of course change them in the regex as well

Reply:
Fixed in 3db8aa41373f3109c7c5c7765c8656e17b571640